src/include/migraphx/op/pooling.hpp

```diff
@@ -20,11 +20,11 @@ namespace op {
 
 struct pooling
 {
-    std::string mode                   = "average";
-    std::array<std::size_t, 2> padding = {{0, 0}};
-    std::array<std::size_t, 2> stride  = {{1, 1}};
-    std::array<std::size_t, 2> lengths = {{1, 1}};
-    padding_mode_t padding_mode        = default_;
+    std::string mode = "average";
+    std::vector<std::size_t> padding{0, 0};
```

Comment:
Is it possible to write this as `std::vector<std::size_t> padding = {0, 0}`? The formatter will align these values.

Reply:
I changed it for the other operators as well.